src/api/BundleDeploy/ParmValidator.ts

```diff
@@ -281,8 +281,32 @@ export class ParmValidator {
       throw new Error("--jobcard parameter is empty");
     }
 
-    // resolve any new line escape sequences embedded in the jobcard
-    params.arguments.jobcard = params.arguments.jobcard.replace("\\n", "\n");
+    // handle long jobcards
```

Comment:
`validateJobcard()` is turning into a bit of a monster, suggest breaking it up into a few smaller functions to improve readability.

Reply:
There's a reasonable chance that I'll want to refactor the code to handle long lines for the bundledir; I don't know whether the algorithm can be shared yet or not, but if it can I'll want to break the associated logic out as part of that work. Will get to that shortly.